dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -431,12 +487,34 @@ def command_for_tests(
         collect_only=collect_only,
         remove_arm_packages=remove_arm_packages,
         github_repository=github_repository,
+        run_db_tests_only=run_db_tests_only,
+        skip_db_tests=skip_db_tests,
+        use_xdist=use_xdist,
+        enable_coverage=enable_coverage,
+        parallelism=parallelism,
+        skip_provider_tests=skip_provider_tests,
     )
     rebuild_or_pull_ci_image_if_needed(command_params=exec_shell_params)
     cleanup_python_generated_files()
     perform_environment_checks()
     if run_in_parallel:
+        if test_type != "All":
```

Comment:
Renamed all the "All" kinds of test types where they just select a subset of tests based on the markers to begin with "All". 

I will add a bit more documentation and example shortly but "All-CI" allows to easily simulate what CI is doing when "full-tests-needed" is used. Basically it will be:

```
breeze testing tests --parallel-test-types All-CI --run-tests-in-parallel --run-only-db-tests
```